tailwind.config.js

```diff
@@ -0,0 +1,5 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  presets: [require('./libs/ui/tailwind.config.js')],
+  darkMode: ['class', '.dark-mode'],
```

Comment:
Why does this file exist:

1. By default, Tailwind uses the prefers-color-scheme CSS media feature to know the current theme, but sometimes we need to manually switch themes, so it is necessary to modify the config to use the class strategy instead of the media strategy.

2. "How to identify the theme" does not seem to be a function of UIKit, but rather a function of the website, so I did not directly modify `./libs/ui/tailwind.config.js`,` but instead created a config specifically for the website.

Reply:
im not checked it yet. if not related to the issue, can we skip this at the moment?

Reply:
@preschian This change is to implement a light/dark mode for the progress bar. Previously, its color was the primary theme color, so there was no issue when switching themes. However, now it is neutral gray, which requires setting different value in dark mode.

In the past, different dark mode styles were managed with `@include ktheme`. But as we shift to using tailwindcss, without these config changes, the ability to switch themes manually won't work properly.

So I think the changes is necessary. But if you're concerned about the risks involved with new config file, I can use the old `@include ktheme` approach for now. What's your opinion?

Reply:
For light/dark mode tailwind colour, refer to this file https://github.com/kodadot/nft-gallery/blob/b9e4ae2fcd397bbc8ed96aa24b9b27a19e1fca49/libs/ui/src/scss/tailwind.scss

With that, we can still switch the colour using tailwind, right? For example: `bg-background-color` or `@apply bg-backgroundcolor` on css/scss

Reply:
Or is it a different case? Can you refer to the code for the progress bar? I will check to make sure.

Reply:
https://github.com/kodadot/nft-gallery/pull/9264/files#diff-c43be1564582c288228eea4b143873a8e01384766cf23f4a206155aa98b8503aR58

something like this:

```vue
<span class="text-neutral-5 dark:text-neutral-9" />
```

I used tailwindcss [dark variant](https://tailwindcss.com/docs/dark-mode) here.

we have neutral gray pairs in [figma](https://www.figma.com/file/JXNtryaNYhY0XJNArzvPQx/Handoff?type=design&node-id=5312-19769&mode=design&t=DatXpkV9JsdoFjGe-0):
![CleanShot 2024-02-04-smiwhTX6@2x](https://github.com/kodadot/nft-gallery/assets/10708802/46491a5f-c193-4e04-8a1c-22976e233474)


Of course we could give every color that changes between light and dark modes its own name, but dark variant seems simpler

Reply:
> But if you're concerned about the risks involved with new config file, I can use the old @include ktheme approach for now.

aahh, I see. Thanks for the explanation 👍🏻 I'm ok with this way, then, instead of still using the old `@include ktheme`

fyi @roiLeo. maybe have another opinion about that